_sass/custom/_blog.scss

```diff
@@ -50,21 +41,25 @@ $post-max-width: 1024px;
     font-size: 18px;
```

Comment:
should you be using here variables for font-size? `$font-title-s` or something like that?

Reply:
@lilyndk please check later, I will merge this and you can resolve next if you think is necessary.